packages/shopify-app-remix/docs/staticPages/billing.doc.ts

```diff
@@ -0,0 +1,89 @@
+import {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const data: LandingTemplateSchema = {
+  id: 'guide-billing',
+  title: 'Billing for your app',
+  description:
+    'You can bill merchants for your app via Shopify using the Admin API, either as a one-time purchase or as a recurring subscription.' +
+    '\n\nShopify will handle the transactions, based on purchases set up by the app and agreed upon by the merchant.' +
+    '\n\nFor more information on how this process works, see [the billing documentation](/docs/apps/billing).',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'plans',
+      title: 'Setting up plans',
+      sectionContent:
+        'Your app can offer one or more plans to merchants. These plans can be one-time purchases or recurring subscriptions.' +
+        '\n\nTo configure a plan, use the `billing` configuration when calling `shopifyApp`. This configuration is a hash, where the keys are the plan name that will appear in the purchase, and the values are the plan settings.',
+      codeblock: {
+        title: '/app/routes/**/*.tsx',
+        tabs: [
+          {
+            title: '/app/routes/**/*.tsx',
+            code: './examples/guides/billing/config.example.tsx',
+            language: 'tsx',
+          },
+        ],
+      },
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'require',
+      title: 'Gating requests',
+      sectionContent:
+        'You can ensure that merchants have paid for the app before allowing them to access certain routes.' +
+        "\n\nTo do this, you can use the `admin.billing.require` function. This function will verify that there is an active payment, or require it if there isn't." +
+        '\n\nYou can pass in multiple plans to `require`. It will pass if there is a purchase for **any** of the plans, and return information on the active purchase.' +
+        "\n\n> Tip: Make sure to call this function in both loaders and actions so you don't end up with ungated requests.\n> You can also call it in a layout to gate multiple routes.",
```

Comment:
Should we show a layout example? It seems like that would be the most likely way folks will be using this.